### ISSUE TITLE
"SERIALIZER MUST BE A NORMALIZER" during export

### ISSUE THREAD

**Issue by ALahmer** [NijiDigital/akeneo-labelized-export-bundle#1]
During export, after upgrade to akeneo 3.0 i have this error:
`16:01:36 ERROR     [batch] Encountered an error executing the step: Serializer must be a normalizer
An error occurred during the export execution.
Error #0 in class LogicException: Serializer must be a normalizer
16:01:36 ERROR     [batch] Encountered an error executing the step: Serializer must be a normalizer
16:01:36 ERROR     [app] Exception occurred while flushing email queue: Connection could not be established with host localhost [No route to host #113]`

I think error is trown in line https://github.com/NijiDigital/akeneo-labelized-export-bundle/blob/6e5b36f99dce910160cc71df9250a0edd7ff1674/Normalizer/Product/Standard/PropertiesNormalizer.php#L39

Is there a way to adapt this bundle to work also with 3.0 version of akeneo?

Thank you

**Comment by bpresles** [comment]
Indeed, this bundle is not yet compatible with Akeneo 3.0. We plan to migrate it for next month.

But if you need it before and are willing to fix it yourself, feel free to do it and open a merge request with your fix.

**Comment by ALahmer** [comment]
Thanks for you answer @bpresles , i will try.

I've noted also an other point that maybe you can manage in the new release, i will report it in a new issue.

Thank you
